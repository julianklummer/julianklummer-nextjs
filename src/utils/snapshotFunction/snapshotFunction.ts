const benderList = [
  { name: 'Katara', power: ['water'] },
  { name: 'Aang', power: ['water', 'fire', 'earth', 'air'] },
  { name: 'Toph', power: ['earth', 'metal'] }
]

export const getEarthBenders = () => {
  return benderList.filter((bender) => bender.power.includes('earth'))
}
