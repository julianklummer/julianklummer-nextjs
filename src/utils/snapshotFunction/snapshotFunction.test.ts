import { getEarthBenders } from './snapshotFunction'

test('getEarthBenders', () => {
  expect(getEarthBenders()).toMatchSnapshot()
})
