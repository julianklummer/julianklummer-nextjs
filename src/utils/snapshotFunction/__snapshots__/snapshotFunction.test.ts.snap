// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getEarthBenders 1`] = `
[
  {
    "name": "Aang",
    "power": [
      "water",
      "fire",
      "earth",
      "air",
    ],
  },
  {
    "name": "Toph",
    "power": [
      "earth",
      "metal",
    ],
  },
]
`;
